### DISCUSSION TITLE
Default page view? (especially random ordering)

### DISCUSSION THREAD

**Discussion by jecorn** [mealie-recipes/mealie#2298] [Q&A]
I'm really loving mealie! But on v1b5 with a lot of recipes, my All Recipes view only shows a little snapshot. Is there a way to show a random set of recipes on a page? The random button is very nice, but I would more often like to show a random set (10-20, maybe user-configurable) and then choose a winner from those.

**Comment by michael-genson** [comment]
The front page is totally revamped in nightly, with a bunch of different sort options (similar to the all recipes page in beta 5, but it's much better now)

I don't think the random function is any different, though

**Comment by jecorn** [comment]
I just tried to log into the demo page to check it out and am getting an invalid credentials error.

**Comment by DrSpaldo** [comment]
Can the sort method be set as default? Ie. I want them sorted alphabetically by default

**Reply by michael-genson** [reply]
If you're on nightly, the sort controls persist through sessions. That means that once you set the sort (e.g. alphabetical) it will stay that way forever (until you change it, of course).

**Reply by DrSpaldo** [reply]
> If you're on nightly, the sort controls persist through sessions. That means that once you set the sort (e.g. alphabetical) it will stay that way forever (until you change it, of course).

Thanks for the reply @michael-genson , I am on the nightly. I thought it was resetting my last view, I will check over the next few days.

**Reply by DrSpaldo** [reply]
@michael-genson I’ve just confirmed it does not remember / retain the view order. Tested in iPhone Home Screen shortcut

**Comment by michael-genson** [comment]
The landing page/sorting tools have changed a bunch since this was opened; if you're still having issues with persistence, feel free to open an issue
